### ISSUE TITLE
Adds falsy `use const False\MyClass\true;` statements

### ISSUE THREAD

**Issue by cracksalad** [eclipse-pdt/pdt#256]
**Describe the bug**

When the setting PHP > Editor > Save Actions > Organize use statements is on and you use `true` in a namespaced class, eclipse/PDT will add a use statement `use const False\MyClass\true;` on save. This use statement can not be resolved and thereby effectivly kills the class.

**Describe the eclipse environment**

Tested in eclipse 2023-06 and 2023-09. PDT 8.0 and 8.1.

**Describe your system**
 - OS Windows
 - Version 10
 - PHP Version 8.2

**To Reproduce**
1. Switch the setting "PHP > Editor > Save Actions > Organize use statements" on
2. Create a namespaced class and use the constant `true` in it
3. Hit save and observe the (namespace) use statements

**Expected behavior**

use statements should generally be added, but not if they are not needed and can not be resolved. 

**Workaround**

1. Disable this setting (I do not want to, I like it)
2. Use the global search (magnifier icon) to replace those statements (using regex) `use const False\\MyClass\\true;\n` with empty string - everytime you change a file.

**Comment by pounard** [comment]
It happens a lot to me, but it's very hard to reproduce. Mostly it happens when I hit Ctrl+S then Ctrl+Shift+O while save is unfinished, or when the file is being validated after some change. It's quite random, arbitrary, it happens in many files.

**Comment by zulus** [comment]
yeah, not only here. Auto generated getter/setter completely ignore FQN's . Probably due same reason

I'll take a look
